fix note for lb in deploy-on-azure-aks.md (#1851)
pingcap/docs-tidb-operator: en/deploy-on-azure-aks.md

> **Note:**
>

> > By default, TiDB LoadBalancer in `tidb-cluster.yaml` is set to "internal", indicating that the LoadBalancer is only accessible within the cluster virtual network, not externally. To access TiDB over the MySQL protocol, you need to use a bastion to access the internal host of the cluster or use `kubectl port-forward`. You can delete the annotation following in the tidb-cluster.yaml file:
> > 
> > annotations:
> > service.beta.kubernetes.io/azure-load-balancer-internal: "true"
> > 
> > to expose the LoadBalancer publicly. However, notice that this practice may expose TiDB to risks.

Reviewer: ```suggestion
> By default, TiDB LoadBalancer in `tidb-cluster.yaml` is set to "internal", indicating that the LoadBalancer is only accessible within the cluster virtual network, not externally. To access TiDB over the MySQL protocol, you need to use a bastion to access the internal host of the cluster or use `kubectl port-forward`. If you understand the risks of exposing the LoadBalancer publicly, you can delete the following annotation in the `tidb-cluster.yaml` file:
> 
> ```yaml
> annotations:
> service.beta.kubernetes.io/azure-load-balancer-internal: "true"
> ```
>
> After deleting the annotation, the recreated LoadBalancer and its associated TiDB services will be externally accessible.
```
